### ISSUE TITLE
Problem with shurcooL/githubql querying

### ISSUE THREAD

**Issue by shuchow** [justwatchcom/github-releases-notifier#1]
Hi, I wanted you to be aware of a possible issue with shurcooL/githubql's package.  Querying for releases is failing a lot.  The output when starting the docker image is:

`{"caller":"releasechecker.go:40","err":"can't find any releases for angular/angular","msg":"failed to query the repository's releases","name":"angular","owner":"angular","severity":"warn","ts":"2017-11-28T22:03:33.95055614Z"}` 

Some of the things I've found:

primefaces/primeng, singularityware/singularity seems to work.  The following are failing: rstudio/rstudio, angular/angular, pallets/flask, zzzeek/sqlalchemy, rstudio/shiny-server 

I haven't investigated githubql package a lot, but it seems like there was a recent (within 24 hours) change to their api.

**Comment by metalmatze** [comment]
Yep. The problem is on GitHubs side. Sadly when someone doesn't create a _GitHub Release_ and only uses git tags it's not a real release for GitHub's API...

**Comment by dominikschulz** [comment]
So you're saying that this is rather the maintainers fault?
In that case we can't do anything about it and close this issue.

**Comment by metalmatze** [comment]
Yes and no, sadly.
We could try to parse or do some sort ordering for the raw git tags, but I don't think that this will result in good notifications in the end. They probably are going spam your channels with wrong tags. Plus we don't really have metadata for those.

I think this clearly made on purpose by GitHub so people use their releases, because they could provide information for when the tag has been created...

I'm not really sure how we can move forward with this. Fortunately I have another project with exactly the same problem. Maybe I might solve this problem there and then port back... but no guarantees.

**Comment by metalmatze** [comment]
Just to be sure I looked at the v3 API. 
The releases provide a lot of information on the release.
https://developer.github.com/v3/repos/releases/#get-the-latest-release

But if there's no release created on GitHub and only a git tag present, we don't get much there either...
https://developer.github.com/v3/repos/#list-tags

It's really bad. I really would like to be able to get a notification e.g. when a new Go release is tagged... :expressionless:

**Comment by dominikschulz** [comment]
IMHO we should rely on provided API and not heuristics, but we could make this optional.
